framework/contracts/account/manager/src/commands.rs

```diff
@@ -297,17 +305,22 @@ pub fn set_owner(
     acc_info.governance_details = verified_gov.clone();
     INFO.save(deps.storage, &acc_info)?;
 
-    // Update the Owner of the Account
-    let ownership = cw_ownable::update_ownership(
-        deps,
-        &env.block,
-        &info.sender,
-        cw_ownable::Action::TransferOwnership {
-            new_owner: new_owner_addr.into_string(),
-            expiry: None,
-        },
-    )?;
-
+    let ownership = if forced {
+        // Force move ownership
+        // Note: this will not require claim
```

Comment:
And this is required because the factory sets the owner after sub-account creation?

Reply:
TL;DR: 
1. We can't do it in manager intantiate because account doesn't have a proxy
2. We can't make account factory owner first and move ownership after that, because it will require claim from owner

Reply:
Yeah for case 2 I think we should come up with a different solution. We **know** that on instantiation the factory is the caller so maybe we can have a `CACHE_ADMIN` or whatever that uses the regular Admin type and is set and used as long as it is set. On its first admin change the cache is cleared and the contracts moves over to the cw-ownable admin. wdyt?

Reply:
Ok, if that's your preference!

Reply:
I like that approach, do you think we check for cached admin first or after cw_ownable didn't match? Thinking to create `remove_cache_admin` method for this case instead of transfer admin after instantiation, thoughts?

Reply:
> @CyberHoward Is there a noticeable downside for adding forced mode to move ownership, that I'm missing?

It's kind of ugly in my opinion (because it changes the contract's API) as it's only a requirement for one specify scenario that can only take place on instantiation.


> I like that approach, do you think we check for cached admin first or after cw_ownable didn't match? Thinking to create `remove_cache_admin` method for this case instead of transfer admin after instantiation, thoughts?

You mean adding a new endpoint to the execute message? I think we can get by without changing the API of the contract.

Reply:
I don't get how to achieve what you propose without changing the API, when do we clear cache admin?

Reply:
Ok, I'm just going to allow account_factory to install modules, it already has power to update module addresses, don't think it creates new vector of attacks